libs/market-depth/src/lib/orderbook-row.tsx

```diff
@@ -50,12 +50,12 @@ export const OrderbookRow = React.memo(
         <PriceCell
           testId={`price-${price}`}
           value={BigInt(price)}
-          valueFormatted={addDecimalsFormatNumber(price, decimalPlaces)}
+          valueFormatted={addDecimalsFormatNumber(price, positionDecimalPlaces)}
         />
         <Vol
           testId={`ask-vol-${price}`}
           value={ask}
-          valueFormatted={addDecimal(ask, positionDecimalPlaces)}
+          valueFormatted={addDecimal(ask, decimalPlaces)}
```

Comment:
chore (blocking): should be market.positionDecimalPlaces for volume/size

Reply:
changed it back